pkg/config/config_test.go

```diff
@@ -414,6 +414,18 @@ var _ = Describe("Config", func() {
 			Expect(config).To(BeNil())
 		})
 
+		It("should succeed and return default containers.conf with libpod.conf", func() {
+			config, err := NewConfig("testdata/libpod_default.conf")
```

Comment:
This test would not be valid, since it would be just reading libod_default.conf as a containers.conf file.